pftools/python/parflow/tools/export.py

```diff
@@ -30,7 +30,7 @@ def _extract_sub_surface_props(self, geom_item):
         entry = {'key': name}
         has_data = False
         for key in self.pfkey_to_alias:
-            value = geom_item.get(key, skip_default=True)
+            value = geom_item.get_value_from_location(key, skip_default=True)
```

Comment:
Is the key notation from `pfkey_to_alias` using the `A.B.C` notation?

Reply:
Actually, it might be a single key and that is why it must be working.

Reply:
Actually that is not true, [see](https://github.com/parflow/parflow/blob/master/pftools/python/parflow/tools/ref/table_keys.yaml), we use the `.` notation

Reply:
The tests that validate that part must just print the table rather than comparing its actual content.

Reply:
This is now being tested, and was fixed.